source/ota_mqtt.c

```diff
@@ -848,7 +821,7 @@ OtaErr_t requestFileBlock_Mqtt( OtaAgentContext_t * pAgentCtx )
     /* Reset number of blocks requested. */
     pAgentCtx->numOfBlocksToReceive = otaconfigMAX_NUM_BLOCKS_REQUEST;
 
-    uint32_t blockSize = OTA_FILE_BLOCK_SIZE & 0x7fffffffU;
+    blockSize = OTA_FILE_BLOCK_SIZE & 0x7fffffffU;
```

Comment:
where does the constant "0x7fffffffU" come from? Can you make a macro for this for readability or add a comment?

Reply:
I don't know where it came from, I was merely fixing compiler warnings by moving the declaration to top. @pvyawaha any idea of this magic number?

Reply:
Looks like it's a legacy workaround, https://github.com/aws/amazon-freertos/blob/be79e3a653bd8acf2ea575662c59e33be0439773/libraries/freertos_plus/aws/ota/src/mqtt/aws_iot_ota_mqtt.c#L848. I'll remove it.